packages/notebook-extension/src/index.ts

```diff
@@ -849,6 +854,22 @@ function addCommands(app: JupyterLab, services: IServiceManager, tracker: Notebo
     },
     isEnabled: hasWidget
   });
+  commands.addCommand(CommandIDs.createConsoleForNotebook, {
+    label: 'Create Console for Notebook',
+    execute: args => {
+      let widget = tracker.currentWidget;
```

Comment:
If you use the `getCurrent()` function here, it will make sure the notebook is focused.